wordpress-mobile/WordPress-Android
# Fix email address goes blank issue

Fixes email field going blank on pressing OK in Account Settings

Fixes #16795 

To test:

- Login to the app
- Go to me -> Account settings
- Click at email address
- Press OK
- Verify that email address field doesn't go blank as before

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
existing tests

PR submission checklist:


UI Changes testing checklist:
